sensor_msgs/NavSatFix gps_position

uint8 nums

float64 latitude_sigma
float64 longitude_sigma
float64 altitude_sigma

float64 northing
float64 easting
float64 down

string utm_zone